packages/dd-trace/src/config.js

```diff
@@ -369,6 +369,11 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
       maybeFile(appsec.blockedTemplateJson),
       maybeFile(process.env.DD_APPSEC_HTTP_BLOCKED_TEMPLATE_JSON)
     )
+    const DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING = coalesce(
```

Comment:
I think we can leave that way, in case the variable is misspelled the mode will default to disabled so the user likely will review the variable. Anyway in case there is a negative feedback on this particular thing we can fix it easily in a future release.

Reply:
Ok sounds good to me :+1: